cpp/log/cert.h

```diff
@@ -8,6 +8,9 @@
 #include <vector>
 
 #include "base/macros.h"
+#include "util/statusor.h"
+
+using util::StatusOr;
```

Comment:
Shouldn't use `using` in headers top-levels:

https://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Namespaces